plugins/infinispan90/src/main/java/org/radargun/service/Infinispan81EmbeddedContinuousQuery.java

```diff
@@ -0,0 +1,20 @@
+package org.radargun.service;
+
+import org.infinispan.Cache;
+import org.radargun.traits.ContinuousQuery;
+
+/**
+ * @author Vojtech Juranek &lt;vjuranek@redhat.com&gt;
+ */
+public abstract class Infinispan81EmbeddedContinuousQuery implements ContinuousQuery {
```

Comment:
Same comment here about naming. I know the code is coming from that plugin, but it could be weird for users.